portfolio.css

```diff
@@ -20,765 +90,601 @@
+  margin-top: 1vh;
+}
 
-  .header-container {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    width: 100%;
-    height: 50px;
-  }
+.light {
+  display: flex;
+  align-items: flex-end;
+  margin-top: 20vh;
+  width: 100%;
+}
 
-  .Nav-1 a {
-    color: #28352f;
-    font-size: 5.5vw;
-    padding-left: 5vw;
+@media (min-width: 991.99px) {
+  .logo {
+    display: none;
   }
 
-  .Nav-2 a {
-    padding-right: 5vw;
+  .menu-icons {
+    display: none;
   }
 
-  .section-title {
-    font-size: 7.2vw;
-    text-align: center;
-    font-family: 'Inter', sans-serif;
-    font-weight: 700;
-    width: 100%;
-    color: #172b4d;
-    line-height: 12vw;
-    margin-top: 21%;
+  header {
+    display: none;
   }
 
-  .social-icons {
-    width: 5vw;
-    height: 6vw;
-    margin-right: 3vw;
-    margin-left: 3vw;
+  #s-1 ul {
+    display: flex;
+    justify-content: space-between;
+    width: 250px;
+    margin: 25px auto;
   }
 
   #s-1 {
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
+    justify-content: flex-start;
     align-items: center;
-    height: auto;
-    text-align: center;
-    background-image: url(images/header-illsutration-mobile.svg);
-    background-size: 100% 100%;
+    background-image: url('images/Header-llustration-desktop.svg');
+    background-size: 100% auto;
+    background-repeat: no-repeat;
     padding: 0% 10% 0% 10%;
     width: 100%;
   }
 
+  .section-title {
+    font-size: 25px;
+    text-align: center;
+    font-weight: 700;
+    width: 100%;
+    color: #172b4d;
+    line-height: 60px;
+    margin-top: 15%;
+  }
+
   .A-con p {
+    background-position-y: bottom;
+    background-size: 20%;
+    background-repeat: no-repeat;
     text-align: center;
     font-family: 'Inter', sans-serif;
-    font-size: 5.4vw;
-    font-weight: lighter;
-    line-height: 7.5vw;
+    font-size: 20px;
+    line-height: 32px;
     flex-wrap: wrap;
     padding-left: 7%;
     padding-right: 7%;
     color: #42526e;
-    margin-top: 5%;
+    margin-top: 24px;
     height: auto;
+    width: 60%;
   }
 
   #s-1 p {
-    font-size: 5.5vw;
+    font-size: 18px;
     font-family: 'Inter', sans-serif;
     font-weight: 400;
     text-align: center;
     color: #172b4d;
-    margin-top: 5vw;
-    margin-bottom: 25%;
-  }
-
-  #s-1 .social-media {
-    display: flex;
-    justify-content: center;
-    width: auto;
-  }
-
-  .card-content-2 ul {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    width: 100%;
     margin-top: 20px;
-    padding-left: 3%;
-    padding-right: 3%;
-    flex-wrap: wrap;
+    width: 65%;
+    margin-bottom: 0%;
   }
 
-  .card-content ul {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-around;
-    width: 100%;
-    margin-top: 3%;
-    margin-bottom: 3%;
-    background: #fff;
+  .social-icons {
+    width: 1.5vw;
+    height: 1.5vw;
+    margin-right: 0.1vw;
+    margin-left: 0.1vw;
```

Comment:
OPTIONAL

- [ ] You have over 900 lines of CSS. Within these lines, so many properties have been repeated.
- Group similar properties used more than twice and then assign that class within the HTML structure
```
    display: flex;
    flex-direction: row;
    justify-content: space-around;
```
- Use the known CSS properties that combine more than one property.
```
margin: 0 0.1vw;
border: 1px solid red;
background: center cover no-repeat;
```

Adapt to these practices to keep your code DRY which is also the best practice of writing code.